src/sentry/static/sentry/app/views/organizationIntegrations/addIntegration.tsx

```diff
@@ -66,10 +83,10 @@ export default class AddIntegration extends React.Component {
     const opts = `scrollbars=yes,width=${width},height=${height},top=${top},left=${left}`;
 
     this.dialog = window.open(installUrl, name, opts);
-    this.dialog.focus();
+    this.dialog && this.dialog.focus();
```

Comment:
This has been a source of sentry errors, maybe see if you can find the issues on Sentry and link their short IDs to the PR

Reply:
Like this?

![Screen Shot 2019-11-22 at 2 06 43 PM](https://user-images.githubusercontent.com/8533851/69463752-5f33a480-0d31-11ea-8e87-f7e00c71a12a.png)

Reply:
No, it's this 
![image](https://user-images.githubusercontent.com/79684/69464088-424ba100-0d32-11ea-8f55-b954ff2da4af.png)